doc/user/standalone-distribution.md

```diff
@@ -72,7 +72,7 @@ Set `TRUFFLERUBY_VERSION` to the latest TruffleRuby version from
 
 ```bash
 $ export TRUFFLERUBY_VERSION=<desired_version>
-$ curl -L https://github.com/oracle/truffleruby/releases/download/vm-$TRUFFLERUBY_VERSION/truffleruby-$TRUFFLERUBY_VERSION-linux-amd64.tar.gz | tar xz
+$ curl -L https://github.com/oracle/truffleruby/archive/$TRUFFLERUBY_VERSION/truffleruby-$TRUFFLERUBY_VERSION-linux-amd64.tar.gz | tar xz
```

Comment:
Thanks for opening a PR!
`archive` is just a zip of the source files though, it doesn't contain the release files.
There is no TruffleRuby builds for 20.1.0, it's just a tag, not a release: https://github.com/oracle/truffleruby/releases

The reason is there was no relevant security change for TruffleRuby, so 20.1.0 would be the same as 20.0.0.
In other words, https://github.com/oracle/truffleruby/releases/tag/vm-20.0.0 is still the latest TruffleRuby release.
Sorry for the confusion.

Reply:
I see, I'll change it back, then.